src/pages/dataElementGroups/fields/CodeField.tsx

```diff
@@ -0,0 +1,25 @@
+import i18n from '@dhis2/d2-i18n'
+import { InputFieldFF } from '@dhis2/ui'
+import React from 'react'
+import { Field as FieldRFF } from 'react-final-form'
+import { useCheckMaxLengthFromSchema } from '../../../lib'
+import type { SchemaName } from '../../../types'
+
+export function CodeField() {
+    const validate = useCheckMaxLengthFromSchema(
```

Comment:
I know the `useCheckMaxLengthFromSchema` wasn't added in this PR. However I just checked the implementation, and due to `as number`-assertion this could cause a runtime error if the length field does not exist. Because the `createMaxCharacterLength` throws an error if undefined. So I think it would be safe to have some kind of fallback, or ignore the length completely in the `validate` function.

I think we could